lib/index.js

```diff
@@ -773,24 +724,24 @@ MaplibreGeocoder.prototype = {
             res.features = res.features.filter(this.options.filter);
           }
 
-          if (res.features.length) {
-            this._clearEl.style.display = "block";
+          var results = [];
+          if (res.suggestions) results = res.suggestions;
```

Comment:
nit: if statements without curly braces can lead to easy editing mistakes. Should we add some linting rules or prettier to the repo?